ctp2_code/ai/strategy/scheduler/scheduler.cpp

```diff
@@ -723,6 +720,7 @@ void Scheduler::Match_Resources(const bool move_armies)
 					--goal_iter;
```

Comment:
That isn't right either. And my feeling is that this code around has some more problems.